docker/pdo_ccf.dockerfile

```diff
@@ -49,11 +49,8 @@ RUN /project/pdo/tools/build_ccf.sh
 # Network ports for running services
 EXPOSE 6600
 
-ARG PDO_HOSTNAME
-ENV PDO_HOSTNAME=$PDO_HOSTNAME
-
-ARG PDO_LEDGER_URL
-ENV PDO_LEDGER_URL=$PDO_LEDGER_URL
+ARG PDO_HOSTNAME=localhost
+ENV PDO_HOSTNAME=${PDO_HOSTNAME}
 
 # Note that the entry point when specified with exec syntax
 # can be extended through the docker run interface far more
```

Comment:
this needs more discussion. adding the hostname and ledger url allows construction of pre-configured images (that is, the image contains the url). in practice, however, its WAY easier to just specify the interface and ledger in the startup script parameters.

Reply:
this is an outdated change regardless of github not marking it as such. In fact, the actual change is exactly the opposite (i.e., exactly what you want?)